### DISCUSSION TITLE
Error to display /admin with new version 1.28.1

### DISCUSSION THREAD

**Discussion by mage0se** [dani-garcia/vaultwarden#3437] [General]
Hello,
I have a problem with version 1.28.1 with an error when I try to connect to the admin, and I don't understand why.

{"ErrorModel":{"Message": "Handlebars", "Object": "error"}, "ExceptionMessage":null, "ExceptionStackTrace":null, "InnerExceptionMessage":null, "Message": "Handlebars", "Object": "error", "ValidationErrors":{"":["Handlebars"]}, "error":"", "error_description":""}

I have this message in the log:
`
[2023-04-11 15:02:02.531][vaultwarden::api::web][ERROR] Static file not found: identicon.js
[2023-04-11 15:02:32.968][request][INFO] POST /admin
[2023-04-11 15:02:32.973][error][ERROR] Handlebars.
[CAUSE] RenderError {
    desc: "Variable "config" not found in strict mode.",
    template_name: Some(
        "admin/settings",
    ),
    line_no: Some(
        12,
    ),
    column_no: Some(
        17,
    ),
    cause: None,
    unimplemented: false,
}`

Could you help me to get the admin console

**Comment by stefan0xC** [comment]
Did you overwrite the admin templates? There have been some changes between 1.27.0 and 1.28.0 so you need to update the handlebar files (by default in `data/templates/admin` but it depends on if you set the `TEMPLATES_FOLDER` or changed the `DATA_FOLDER`).

**Comment by mage0se** [comment]
Hello @stefan0xC,
Thank you very match for your help
Yes, you are right I had forgotten to put the templates ;( ... sorry, dumb error ...

But yet i have a other error when i would like use admin/users : Refused to execute script from 'https://portal1.vault.she.safran/vw_static/jquery-3.6.4.slim.js' because its MIME type ('application/json') is not executable, and strict MIME type checking is enabled in web application. 

You can perhaps this error ? Thand you.

**Comment by mage0se** [comment]
I have find the problem. The last version template (master) call https://portal1.vault.she.safran/vw_static/jquery-3.6.4.slim.js library, but in version 1.28.1, template users is jquery-3.6.3.slim.js and not jquery-3.6.4.slim.js.
I think my upgrade version is OK, but complicated this time !
I did not take into account: ADMIN_TOKEN which is insecure. But it doesn't make me want to modify :'(
